salt/_pillar/metalk8s_private.py

```diff
@@ -28,6 +30,23 @@ def _read_sa_private_key():
     return {"sa_private_key": key_data}
 
 
+def _read_apiserver_key():
+    # FIXME: we only have access to this file because:
+    #   1- Salt master Pod mounts all of /etc/kubernetes
```

Comment:
You are talking about `/etc/kubernetes` but the key file is stored under `/etc/metalk8s`.

Reply:
Oof, indeed. The remark still holds, edited the path :)